modules/db/src/test/java/org/mule/module/db/integration/select/SelectStreamingExceptionTestCase.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.module.db.integration.select;
+
+import static java.lang.Boolean.FALSE;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.mule.DefaultMuleMessage;
+import org.mule.api.MuleException;
+import org.mule.api.MuleMessage;
+import org.mule.api.client.LocalMuleClient;
+import org.mule.module.db.integration.AbstractDbIntegrationTestCase;
+import org.mule.module.db.integration.model.AbstractTestDatabase;
+import org.mule.module.db.integration.model.DerbyTestDatabase;
+import org.mule.module.db.internal.domain.database.DbConfig;
+import org.mule.module.db.internal.resolver.database.DbConfigResolver;
+import org.mule.transport.NullPayload;
+
+import java.util.Collections;
+import java.util.List;
+
+import javax.sql.DataSource;
+
+import org.junit.Test;
+import org.junit.runners.Parameterized;
+
+public class SelectStreamingExceptionTestCase extends AbstractDbIntegrationTestCase
+{
+
+    public SelectStreamingExceptionTestCase(String dataSourceConfigResource, AbstractTestDatabase testDatabase)
+    {
+        super(dataSourceConfigResource, testDatabase);
+    }
+
+    @Parameterized.Parameters
+    public static List<Object[]> parameters()
+    {
+        return Collections.singletonList(new Object[] {"integration/config/derby-pooling-db-config.xml", new DerbyTestDatabase()});
```

Comment:
In this case i need a pool so i have a certain number of iterations to run before the test fails.